Source/WebCore/bindings/js/JSEventTargetCustom.cpp

```diff
@@ -62,15 +62,15 @@ EventTarget* JSEventTarget::toWrapped(VM&, JSValue value)
     return nullptr;
 }
 
-std::unique_ptr<JSEventTargetWrapper> jsEventTargetCast(VM& vm, JSValue thisValue)
+JSEventTargetWrapper jsEventTargetCast(VM& vm, JSValue thisValue)
 {
     if (auto* target = jsDynamicCast<JSEventTarget*>(thisValue))
-        return makeUnique<JSEventTargetWrapper>(target->wrapped(), *target);
+        return JSEventTargetWrapper { target->wrapped(), *target };
```

Comment:
I think we don’t need the type name in these `return` statements since this isn't using `optional`.